canonical/ubuntu.com
# Checkout visual changes

## Done

- Update checkout redesign to look like https://miro.com/app/board/uXjVONNsexk=/?openComment=3458764546051510037

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- go to /pro/subscribe
    - select any product
    - proceed to checkout
- Checkout should look like https://miro.com/app/board/uXjVONNsexk=/?openComment=3458764546051510037 and accommodate all the changes suggested

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
